packages/react-components/react-dialog/src/stories/DialogDefault.stories.tsx

```diff
@@ -17,7 +17,7 @@ export const Default = () => {
             cumque eaque?
           </DialogBody>
           <DialogActions>
-            <DialogTrigger>
+            <DialogTrigger action="close">
```

Comment:
this might be just my personal opinion, but it feels like it is very easy to forget adding `actioin="close"` and very hard to spot that something is broken.

I think having something like `<DialogClose>` instead of `<DialogTrigger action="close">` would be easier to understand. Closing is something different than triggering.

Reply:
An alternative would be to not make the `close` required, and instead inferring the `action` by the triggers location, if it's inside the `DialogSurface` than it's probably a `close`, if it's outside than it's probably `open`